social-commerce/src/Component/Onboarding/Login/Login.tsx

```diff
@@ -4,16 +4,22 @@ import "./Login.css";
 import { useNavigate } from "react-router-dom";
 
 type FormValues = {
```

Comment:
take the type and add it to the type.check.ts file i created to maintain clean code and follow the format that you will see inside the file just like what me and kennedy did